app/controllers/user.js

```diff
@@ -129,3 +129,65 @@ const emptyToNull = (input) => {
   return newInput;
 
 };
+
+const newUser = (req, res, next, input) => {
```

Comment:
In the merge or rebase you lost the changes where in the normal user creation you used this function.